deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -39,16 +39,6 @@ rules:
   verbs:
   - get
   - list
-### START - view ClusterVersion
-- apiGroups:
-  - config.openshift.io
-  resources:
-  - clusterversions
```

Comment:
Removing `clusterversion` as it will be covered under (line 168):

```
- apiGroups:
  - config.openshift.io
  resources:
  - '*'
  verbs:
  - get
  - list
  - watch
```

Reply:
Could rbac specific to get,list oauths be removed too?

Reply:
yes definitely. I had removed it in my first PR but removing that messes up the _git diff_ and that's what made it hard to read/review last time. I will raise separate small PR for it after this is merged. I have added it as a comment to remind myself in the Jira card.